engine/src/flutter/shell/platform/android/platform_view_android_delegate/platform_view_android_delegate.cc

```diff
@@ -37,6 +37,104 @@ void putStringAttributesIntoBuffer(
   }
 }
 
+int64_t flagsToInt64(flutter::SemanticsFlags flags) {
```

Comment:
Since is doesn't make sense for flags to have a sign, should we use an unsigned type, e.g. `uint64_t`?

Reply:
we will read it on java side using `buffer.getLong();` and it seems there's no unsigned long in java.  So I think it's ok to just use int64 here

Reply:
makes sense to me